tools/container/runtime/containerd/containerd.go

```diff
@@ -164,3 +167,19 @@ func (o *Options) runtimeConfigOverride() (map[string]interface{}, error) {
 
 	return runtimeOptions, nil
 }
+
+func GetLowlevelRuntimePaths(hostRoot string) ([]string, error) {
+	cfg, err := containerd.New(
+		containerd.WithConfigSource(containerd.CommandLineSource(hostRoot)),
+	)
+	if err != nil {
+		return nil, fmt.Errorf("unable to load containerd config: %w", err)
+	}
+	var runtimes []string
+	runtimes = append(runtimes, "runc")
```

Comment:
Just a nit, but the ordering here is different than for `crio`.

Reply:
It is also starting to feel as if there are multiple places where we're specifying this now.

Happy to have use look into this in a follow-up though.

Reply:
The difference in ordering is intentional